gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -39,6 +45,8 @@ type NotificationSettingsTable = {
 })
 export class NotificationsListComponent implements OnInit {
   public notificationsSettingsTable: NotificationSettingsTable[] = [];
+  public newNotificationTemplate: any = [];
```

Comment:
🙏 Please avoid Any. and i think this method is not needed no ?

Reply:
Done